etna/ciim/constants.py

```diff
@@ -56,27 +56,22 @@ class Bucket:
     aggregations: List[str] = field(default_factory=list)
 
     @property
-    def get_normalised_aggregations(self) -> List[str]:
+    def aggregations_normalised(self) -> List[str]:
         """
-        In the API response, the items with the highest number of matches are
-        included for each aggregation. Those values are used to indicate
-        counts for each 'bucket', and to update the form field choices, so that
-        the most relevant filter options are shown.
-        Ex: ["group:30", "catalogue:10",]
+        Returns the a list of strings to include as the 'aggregations' option value when assembling
+        an API request for this bucket. Each string must be in the format "filter-name:item-count".
         """
         values = []
         for aggregation in self.aggregations:
-            item_count = 10
-            if aggregation == Aggregation.GROUP:
-                # Fetch more 'groups' so that we receive a counts
-                # for any bucket/tab options we might be showing
-                # (not just the 10 most popular)
-                item_count = 30
-            values.append(f"{aggregation}:{item_count}")
+            bits = aggregation.split(":")
+            if len(bits) == 2:
+                values.append(bits[0] + ":" + bits[1])
+            else:
+                values.append(bits[0] + ":10")
         return values
 
     def __post_init__(self):
-        self.aggregations = self.get_normalised_aggregations
+        self.aggregations = self.aggregations_normalised
```

Comment:
It's nice that you're thinking about the API here, but it isn't important that this value be normalised at this point in time. It's fine for the value to be lazily generated when it is actually required (you'll see this pattern a lot in Django)

Reply:
It's only as harmful as any other non-optimal code... Having this here means more code for you to test and maintain, and the module probably takes a few more microseconds to initialise. It's just a better design to put off 'work' until it needs doing.

Reply:
I have gone with your suggestions. Can you please clarify what you'd like to do over here, or maybe can you take this as separate ticket?